include/rppt_tensor_arithmetic_operations.h

```diff
@@ -47,8 +47,8 @@ extern "C" {
  *          It multiplies each element of the source tensor by a corresponding element in the 'mulTensor',
  *          adds a corresponding element from the 'addTensor', and stores the result in the destination tensor.
  *          Support added for f32 -> f32 dataype.
- * \image html input150x150x4.gif Sample Input
- * \image html arithmetic_operations_fused_multiply_add_scalar_150x150x4.gif Sample Output
+ * \image html https://raw.githubusercontent.com/ROCm/rpp/develop/docs/data/doxygenInputs/input150x150x4.gif Sample Input
```

Comment:
@randyh62  How come we don't need to provide the whole link for the remaining 8 header files with png images, but only for the ones with gif images?
I see that its now displaying the gifs in the CI docs build though